sencha-workspace/SlateAdmin/app/controller/progress/interims/Report.js

```diff
@@ -310,21 +347,26 @@ Ext.define('SlateAdmin.controller.progress.interims.Report', {
 
     onSectionNotesSaveBtnClick: function() {
         var sectionNotesForm = this.getSectionNotesForm(),
-            sectionNotes = sectionNotesForm.getRecord();
+            sectionDataRecord = sectionNotesForm.getRecord(),
+            section = this.getProgressInterimsSectionsStore().getById(sectionDataRecord.get('SectionID'));
 
-        sectionNotesForm.updateRecord(sectionNotes);
+        if (!section) {
+            return;
+        }
 
-        if (!sectionNotes.dirty) {
+        sectionNotesForm.updateRecord(sectionDataRecord);
+
+        if (!sectionDataRecord.dirty) {
             return;
         }
 
         sectionNotesForm.setLoading('Saving notes&hellip;');
-        sectionNotes.save({
-            callback: function(sectionNotes, operation, success) {
+        sectionDataRecord.save({
+            callback: function(sectionData, operation, success) {
```

Comment:
@nbey call this `savedSectionTermDataRecord`. It should be the same instance as `sectionTermDataRecord` but we need to receive the param and linters don't like the same variable being redeclared. Theoretically the save method COULD return a new model instance, so all the code within this callback should use the potentially-new `saved*` instance reference